selfservice/strategy/oidc/provider.go

```diff
@@ -85,6 +86,7 @@ func UpstreamParameters(provider Provider, upstreamParameters map[string]string)
 		"login_hint": {},
 		"hd":         {},
 		"prompt":     {},
+		"auth_type":  {},
```

Comment:
Would it be too confusing to just translate `prompt=login` to `auth_type=reauthenticate` for Facebook? Seems like, we're adding too many provider specific things.

Reply:
I'm not sure I like provider-specific magic here. I think it's best to defer this to the user.

Reply:
I'd say let's defer it to the user as well.